comp/core/status/statusimpl/status_api_endpoints.go

```diff
@@ -48,13 +44,42 @@ func (s *statusImplementation) getStatus(w http.ResponseWriter, r *http.Request,
 		buff, err = s.GetStatus(format, verbose)
 	}
 
+	if len(buff) != 0 {
+		// scrub status output
+		s := scrubber.DefaultScrubber
+		var e error
+		if format == "json" {
+			buff, e = s.ScrubJSON(buff)
+		} else {
+			buff, e = s.ScrubBytes(buff)
+		}
+		if e != nil {
+			buff = []byte("[REDACTED] - failure to clean the message")
+		}
+	}
```

Comment:
I believe this block should be moved after the check of `err != nil`. Since we are first if there was an error when fetching the status, if there is no error fetching the status we then scrub it and return the value. 

It seems that logic prior to this change was returning the `buff` as part of the error message which might lead to the decision of keeping as the error message, but I think is best to just return an error message without the `buff` information